src/stylesheets/core/mixins/_sidenav.scss

```diff
@@ -3,8 +3,8 @@
 
   > li {
     background-color: transparent;
-    border-top: 1px solid $color-gray;
-    font-size: $h4-font-size;
+    border-top: units(1px) solid color('base-dark');
```

Comment:
I would say we probably don't _need_ to call `units()` on 1px and 2px. But maybe it's just better to be consistent and use it every time?

Reply:
Yeah that was my assumption, it's more verbose but at least it's consistent.

Reply:
we'll probably want to do a find/replace at some point to make everything consistent